singer/transform.py

```diff
@@ -2,6 +2,9 @@
 import decimal
 import logging
 import re
+
+# TODO: Deprecated jsonschema API. Migrate to new one as described in docs:
+# https://python-jsonschema.readthedocs.io/en/v4.22.0/referencing/#migrating-from-refresolver
```

Comment:
Is it going to be done in the next PRs?

Reply:
No. The API is deprecated but still present so we can use the original code without migration to save time.